frontend/src/containers/Notification.js

```diff
@@ -5,12 +5,25 @@ import resetNotifications from '../actions/notification/reset';
 import Icon from '../components/Icon';
 
 class Notification extends Component {
```

Comment:
I don't understand the distinction between the concepts of "closed" and "hide", if it's the same thing then would be better to use a single term